### ISSUE TITLE
One or more errors occured: A task was cancenlled.

### ISSUE THREAD

**Issue by meguygirl** [lay295/TwitchDownloader#1050]
### Checklist

- [X] I have checked the [issue page](https://github.com/lay295/TwitchDownloader/issues?q=is%3Aissue) for duplicates
- [X] I am running the latest version ([download here](https://github.com/lay295/TwitchDownloader/releases/latest))

### Edition

Command Line Interface

### Describe your issue here

For Macos commnad line: After downloading the chat json, the command to download the mp4 from json returns the error: Unhandled exception. System.AggregateException: One or more errors occurred. (A task was canceled.)
<img width="713" alt=" 000" src="https://github.com/lay295/TwitchDownloader/assets/165354957/0d274677-47f9-4c46-b39e-cb7a5b059244">
 This is a new problem whereas before following the steps and entering the commands were successful in downloading chat as mp4

**Comment by ScrubN** [comment]
Weird, the inner stack trace doesn't say what caused the exception.

The cause was almost certainly an HTTP request though. Try disabling 7TV emotes (7TV causes many issues), or use the `--offline` flag.

**Comment by meguygirl** [comment]
Yeah I tried like 3 more times and got the same error and randomly just tried again and it's downloading now. So it looks like it's working now. I don't know what could have been the cause but it looks like it's gone.

**Comment by ScrubN** [comment]
To me, this issue sounds a lot like #723. I'm continually working on increasing the amount of verbose logging in the core library. Hopefully in the future it will enable us to figure out the source of these issues.
